packages/react-i18n/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@m6web/react-i18n",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**🔨 issue:** ‏to revert (set with lerna)